doc/sidecar_configmap.md

```diff
@@ -0,0 +1,203 @@
+      command: ["sh", "-c", "mkdir -p /tmp/scripts; cp -R /scripts/* /tmp/scripts/; mkdir -p /var/lib/tikv/fuse-data"]
+    containers:
+    - name: chaosfs
+      image: pingcap/chaos-fs
+      imagePullpolicy: Always
+      ports:
+      - containerPort: 65534
+      securityContext:
+        privileged: true
+      command:
+        - /usr/local/bin/chaosfs
+        - -addr=:65534
+        - -pidfile=/tmp/fuse/pid
+        - -original=/var/lib/tikv/data
+        - -mountpoint=/var/lib/tikv/fuse-data
+      volumeMounts:
+        - name: tikv
+          mountPath: /var/lib/tikv
+          mountPropagation: Bidirectional
+    volumeMounts:
+    - name: tikv
+      mountPath: /var/lib/tikv
+      mountPropagation: HostToContainer
+    - name: scripts
+      mountPath: /tmp/scripts
+    - name: fuse
+      mountPath: /tmp/fuse
+    volumes:
+    - name: scripts
+      emptyDir: {}
+    - name: fuse
+      emptyDir: {}
+    postStart:
+      tikv:
+        command:
+          - /tmp/scripts/wait-fuse.sh
+```
+> For more sample ConfigMap files, see [examples](https://github.com/pingcap/chaos-mesh/tree/master/examples/chaosfs-configmap)
+
+Description of `chaofs-tikv.yaml`: 
+
+* **name**: defines the name of the sidecar config, this name should be unique across all sidecar configs.
+* **selector**: is used to filter pods to inject sidecar.
+* **initContainers**: defines the [initContainer](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) need to be injected.
+* **container**: defines the sidecar container need to be injected. 
+* **volumeMounts**: defines the new volumeMounts or overwrite the old volumeMounts of the each containers in target pods.
+* **volume**: defines the new volumes for the target pod or overwrite the old volumes in target pods.
+* **postStart**: called after a container is created first. If the handler fails, the containers will failed. 
+Key defines for the name of deployment container. Value defines for the Commands for stating container.
+
+### Containers
+
+#### chaosfs
+
+`chaosfs` container is designed as a sidecar container and [chaosfs](https://github.com/pingcap/chaos-mesh/tree/master/cmd/chaosfs) process runs in this container.
+`chaosfs` uses [fuse libary](https://github.com/hanwen/go-fuse) and [fusermount](https://www.kernel.org/doc/Documentation/filesystems/fuse.txt) tool to implement a fuse-daemon service 
+and mounts the application's data directory. `chaosfs` will hijack all the file system IO cations of the application, so it can be used to simulate various IO fault that often occur in the real world. 
+
+The following config will inject `chaosfs` container to target pods and will start a `chaosfs` process in this container.
+In addition, `chaosfs` container should be run as `privileged` and the [mountPropagation](https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation) 
+field in `chaosfs` Container.volumeMounts should be set to `Bidirectional`. 
```

Comment:
Could you add some comments to explain the detailed reason why we should set the `Bidirectional` value rather than put a link here?